src/Makefile.am

```diff
@@ -44,8 +44,8 @@ libqatzip_la_SOURCES = \
                        qatzip_lz4.c \
                        xxhash.c
 libqatzip_la_CFLAGS = \
-                      -I./ \
-                      -I../include/ \
+                      -I$(srcdir) \
+                      -I$(top_srcdir)/include \
```

Comment:
Because the relative path is wrong. From `QATzip/build/src` + `../include` you only get to `QATzip/build/include` and the headers are not there. The relative paths are relative to where the compiler is launched, not to where the file is.

qatlib needed a similar fix, see intel/qatlib#57

Reply:
hello, @thiagomacieira, very thanks for your PR, but this issue is caused by the script "_cd build .....  ${MAKE-make} -C src V=1_", it means you are build the qatzip lib out the "QZ_ROOT". 
For now, we don't recommand the customer to build the qatzip lib out of the "QZ_ROOT", because it may cause the Version confusion or other issue when you use the perf script from the qatzip. we recommand you to build the qatzip in `QZ_ROOT`, then maybe install the qatzip lib. it would aviod this issue.

Reply:
I don't care about perf scripts. I just need to build and building out-of-tree is a requirement. This patch fixes the build.